README.md

```diff
@@ -12,24 +12,29 @@ previously populated database.  You additionally can use Full-Text Search to per
 The steps below assume you are running a standalone couchbase instance running kv, indexing, fts (in Couchbase 4.5 or
 later) and query services on the same server where the node application will also be running.
 
- 1. Install a Couchbase Server, and start it.
+ 1. Install a Couchbase Server, start it, and load the `travel-sample` bucket.
+    * If you want to use Full-Text Search, set up an FTS index called `hotels` for all documents of type `hotel`
```

Comment:
This might be out of scope here but details on how to do this would be great.

Reply:
Yeah I had to look at the docs on how to create FTS index myself, will try to add more information on this, still new to it all! 👍